Use ProfileWithInlinePolicies for controls in profiles

jan-cerny opened the pull request:
In this format, the items of policies lists are tuples that consists
of a policy ID and control IDs separated by a colon.

jan-cerny committed:
Use ProfileWithInlinePolicies for controls in profiles

In this format, the items of policies lists are tuples that consists
of a policy ID and control IDs separated by a colon.

yuumasato commented on rhel8/profiles/abcd-low-inline.profile:
```diff
@@ -0,0 +1,10 @@
+  - security_patches_up_to_date
+  - abcd:R1
+  - abcd:R2
+  - abcd:R3
```
I don't have a strong opinion, but I have slight preference on this one.

matejak replied:
What I like about it is that it allows to keep overrides of unwanted selections close to the control that introduces them, and it gives user greater freedom with position of security control selections in the file, especially if they come from more than one policy. Do we like the colon as a delimiter though?

yuumasato replied:
The colon looks fine for now.

matejak committed:
Added policy content for ANSSI minimal.

and updated the profile accordingly.

matejak committed:
Improved the ANSSI policy-profile integration.

Added the ANSSI minimal profile to the list of stable profiles.

matejak committed:
Implemented concept of levels.

Policies, controls and profiles can have a level assigned to them.
Low-level controls are supposed to be essential and always be present in a profile,
while the higher-level controls may be used only in strict profiles.

matejak committed:
Ported the ANSSI intermediary profile to policies/controls.

matejak committed:
Added support for notes in controls.

yuumasato committed:
Expand and discuss first 3 ANSSI requirements

matejak committed:
Merge pull request #5 from yuumasato/extend_3_first_anssi_reqs

Expand and discuss first 3 ANSSI requirements

matejak committed:
Renamed ANSSI policy file to more appropriate anssi.yml

matejak committed:
Updated the profile titles from NT to BP

NT refers to the outdated version 1.1, whereas BP is for 1.2.
There is no need to target 1.1

matejak committed:
Increase controls handling robustness.

Don't parse other files than those that end by .yml

yuumasato committed:
Discuss and expand selection of rules for ANSSI R5

ANSSI R5 is about defense in-depth, which is broad topic.
Note a few aspect of defense in-depth and add a few related rules.

jan-cerny committed:
Merge pull request #11 from matejak/profile_with_inline_policies

Added policy content for ANSSI minimal.

matejak commented:
We like prefer the "Inline" to "Separated", and this PR doesn't have any open issues, so I advise to merge it.
